client/Components/Dogs/Dogs.jsx

```diff
@@ -25,14 +26,29 @@ class Dogs extends Component {
 
     return (
       <div>
+        <div>
         <FilterSideBar updateBreed={this.updateBreed} breedId={breedId} />
-        {dogsToDisplay.map(dog => (
-          <div key={dog.id}>
-            <Link to={`${DOGS}/${dog.id}`}>{dog.name}</Link> {dog.price}{' '}
-            {dog.description}
+    <div id = 'dogs'>
+    {dogsToDisplay.map(dog => (
+      <div id ='dog' key={dog.id} >
+      <h1><Link to={`${DOGS}/${dog.id}`}>{dog.name}</Link></h1>
+        <div id='dogblock'>
+          <div id ='dogtext'>
+          Price: ${dog.price}<br/>
+          Gender: {dog.gender}<br/>
+          Age: {dog.age} {dog.ageUnit}s<br/>
+          Size: {dog.size}<br/>
+          Breed: {dog.breed.name}
+            <p id ='description'>{dog.description}</p>
```

Comment:
This is more just a question. So the only difference between dog list and dog view right now is the `description` right?

Reply:
that is a good question. i didnt know which info is appropriate to show on dog list, i will chop off any extras u guys think shouldnt be there

Reply:
actually they both have pretty much the same info so i can def remove some of it from list

Reply:
i'm good with this one... just mentioning that currently that's the only difference. right now having all the info there (except description makes it easier to test filter)

Reply:
they both have the description..if u go to dog view look on the card itself, right below the picture - there is  a description

Reply:
maybe we just add photo, name, breed, and add to card per dog on dog list view.

Reply:
Ok. I will refactor the dog block accordingly so codeclimate also isnt mad